src/main/java/de/tum/in/www1/exerciseapp/domain/MultipleChoiceQuestion.java

```diff
@@ -41,17 +43,58 @@ public MultipleChoiceQuestion answerOptions(List<AnswerOption> answerOptions) {
     public MultipleChoiceQuestion addAnswerOptions(AnswerOption answerOption) {
```

Comment:
please add JavaDoc comments to automatically generated methods as well, in case you change them